docs/rules.md

```diff
@@ -784,7 +786,7 @@ Please refer to [patch-sku-property.md](./patch-sku-property.md) for details.
 ### PathCharacters
 
 Path should contain only recommended characters.
-The recommended characters are 0-9, A-Z, a-z, -, ., _, ~, and :.
+The recommended characters are 0-9, A-Z, a-z, -, ., \_, ~, and :.
```

Comment:
>z, -, ., \_, ~, and  [](http://example.com/codeflow?start=43&length=20)

is this related to your change?

Reply:
No, I havent modified them.
I ran 
```
rush regen-ruleindex

```
and I think it pulls changes from individual .md files